apps/website/src/pages/docs/avatar.mdx

```diff
@@ -13,73 +13,53 @@ import * as AllExamples from 'examples';
 
 <p>{frontmatter.description}</p>
 
-<LiveExample src='Avatar.main.tsx'>
-  <AllExamples.AvatarMainExample client:load />
+<LiveExample src='Avatar.initials.tsx'>
+  <AllExamples.AvatarInitialsExample client:load />
 </LiveExample>
 
-The avatar is an essential UI component, making it easier to identify a person or organization by their photo, icon, or initials. A user’s photo helps put a face to the name and makes it more personal. Any time an app requires a user profile, an avatar should be included. It also makes it very obvious to the user that they are logged in, as the non-logged in state displays differently.
+The avatar is an essential UI component, making it easier to identify a person or organization by their photo, icon, or initials.
 
-## Appearance
+## Usage
+
+Different properties can be passed to the `<Avatar>` component to customize it according to the user's needs.
+
+- `title` prop can be used to specify the name or title associated with the avatar.
+- `abbreviation` prop allows users to provide a short abbreviation or initials for the avatar.
+- `image` prop can be passed to set a custom image source or icon that will be displayed within the avatar.
 
 ### Sizes
 
-There are four sizes of avatars:
+Avatars can be displayed in various sizes, including `small`, `medium`, `large`, and `x-large`. These options offer users the flexibility to choose the most appropriate avatar size.
```

Comment:
Just a thought (optional suggestion). The second sentence seems unnecessary because it can be implied?